contracts/pol/LiquidityManagerCore.sol

```diff
@@ -316,6 +336,10 @@ abstract contract LiquidityManagerCore is ERC721TokenReceiver {
 
         // Get current pool sqrt price
         (uint160 sqrtPriceX96,) = _getPriceAndObservationIndexFromSlot0(pool);
+        // Check for zero value
+        if (sqrtPriceX96 == 0) {
+            revert ZeroValue();
+        }
```

Comment:
Since the function does not revert - need to check for value, which can never be zero.